### PR TITLE
feat: detects eval calls

### PR DESCRIPTION
**Pull request by lee-tammy**

### CODE REVIEW

File: src/analysis.ts [hunk 1]
```diff
@@ -200,3 +231,15 @@ function getPosition(node: Node): Position {
...
+
+/**
+ * Creates a Point of Interest(poi) object
+ *
+ * @param type the type of poi
+ * @param file the file in which the poi is located
+ * @param position the location in the file that the poi is located
+ */
+function createPOI(
+    type: string, file: string, position: Position): PointOfInterest {
```
Comment (kjin) [inline src/analysis.ts]: Nit: just call `file` `fileName` to make the next line simpler.

File: test/test-analysis.ts [hunk 2]
```diff
@@ -157,3 +157,19 @@ test(
...
+
+test('getEvalCalls should return pois for standard eval() usages', async t => {
+  const content = 'const e = eval(doSomethingBad())';
+  const result = analysis.getEvalCalls(content, 'file');
+  t.deepEqual(result.length, 1);
+  t.deepEqual(result[0].type, 'Eval Call');
+});
+
+test(
+    'getEvalCalls should return pois for eval identifiers under aliases',
```
Comment (kjin) [inline test/test-analysis.ts:169]: What if `eval` is passed to a function that invokes it?

File: src/analysis.ts (hunk 1 above)
Comment (lee-tammy) [reply src/analysis.ts]: done

File: test/test-analysis.ts (hunk 2 above)
Comment (lee-tammy) [reply test/test-analysis.ts:169]: Added a test for this

**Review by kjin — APPROVED** [review]

**Pull request merged**